blog_notebooks/nlp/show_me_the_word_count_gutenberg/show_me_the_word_count_gutenberg.ipynb

```diff
@@ -997,6 +986,23 @@
     "\n",
     "Don't want to wait for the next release to use upcoming features? You can download our nightly containers from [Dockerhub](https://hub.docker.com/r/rapidsai/rapidsai-nightly) or install via [Conda](https://anaconda.org/rapidsai-nightly) to stay at the tip of our development branch."
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Other Examples of NLP workflows:\n",
+    "- https://github.com/rapidsai/tpcx-bb/tree/master/tpcx_bb/queries/q17\n",
```

Comment:
```suggestion
    "- [tpcx-bb query 17](https://github.com/rapidsai/tpcx-bb/tree/master/tpcx_bb/queries/q17)\n",
```

Reply:
Same for the other query impl links

Reply:
Should be resolved by: https://github.com/rapidsai/notebooks-contrib/pull/300/commits/cf149e05c2236ba00c399dc9a8c136eac49f7f0f